Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANSILLReduction.py

```diff
@@ -302,6 +302,73 @@ def _normalise(self, ws):
         # masking however is needed to get more reasonable scales in the instrument view
         MaskDetectors(Workspace=ws, DetectorList=[monID, monID+1])
 
+    def _get_vertical_grouping_pattern(self, ws):
+        """
+        Provides vertical grouping pattern and crops to the main detector panel where counts from the beam are measured.
+        :param ws: Empty beam workspace.
+        """
+        inst_name = mtd[ws].getInstrument().getName()
+        min_id = 0
+        if 'D11' in inst_name:
+            step = 128
```

Comment:
D11 is 256x256, D11lr is 128x128, the back panel of D11B is 192x256, please double check you get this right.

Reply:
Correct, D11 number needed changing, since the step should've been 256.